spec/models/category_spec.rb

```diff
@@ -4,34 +4,35 @@
 
   it { is_expected.to have_many(:videos) }
   #it { should have_many(:videos) }
+  it { is_expected.to validate_presence_of(:name) }
 
   describe "#recent_videos" do
 
-    let(:comedies) { Category.create(name: "comedies") }
+    let(:comedies) { Category.create!(name: "comedies") }
```

Comment:
Currently you are using `Category.create....` I pointed out that you should use `Fabricate(:category ...)`. It carries some advantages with it. Creating objects is centralised at 1 spot - Fabricator and if there are changes in requirements for validations you don't have to go and fix each single spec.